### PR TITLE
Fix #83940

### LINKED ISSUE #83940: "Open with Code" context menu is not localized
The context menu "Open with Code" is not localized on my German Windows 10 Pro x64 system. A screenshot is attached showing the Desktop context menu, but the entry is in English for files and regular directories as well.

image

Related to #11804.

Version: 1.39.2 (user setup)
Commit: 6ab598523be7a800d7f3eb4d92d7ab9a66069390
Datum: 2019-10-15T15:35:18.241Z
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
Betriebssystem: Windows_NT x64 10.0.18362

### PR DESCRIPTION
**Pull request by hadzhiyski**
Supported languages: de, en, es, fr, hu, it, pt-br, ru

This PR fixes #83940

### CODE REVIEW

File: build/win32/i18n/messages.en.isl [hunk 1]
```diff
@@ -5,4 +5,5 @@ AssociateWithFiles=Register %1 as an editor for supported file types
 AddToPath=Add to PATH (requires shell restart)
 RunAfter=Run %1 after installation
 Other=Other:
-SourceFile=%1 Source File
\ No newline at end of file
+SourceFile=%1 Source File
+OpenWithCodeContextMeny=Open w&ith %1
```
Comment (jaqra) [inline build/win32/i18n/messages.en.isl]:
```suggestion
OpenWithCodeContextMenu=Open w&ith %1
```

**Comment by jaqra** [comment]
https://github.com/microsoft/vscode/blob/89659aa4b3fa350e948f3bbc54caf9da8a78be33/build/win32/i18n/messages.tr.isl#L8

You can add

```
OpenWithCodeContextMeny=%1 İle Aç
```

But i am not sure which key must be set as shortkey

**Comment by hadzhiyski** [comment]
> https://github.com/microsoft/vscode/blob/89659aa4b3fa350e948f3bbc54caf9da8a78be33/build/win32/i18n/messages.tr.isl#L8
> 
> You can add
> 
> ```
> OpenWithCodeContextMeny=%1 İle Aç
> ```
> 
> But i am not sure which key must be set as shortkey

Thanks. I will try to research what is the correct translation for all supported languages

**Comment by jaqra** [comment]
It is correct. It is my main language. You can also see from

https://github.com/microsoft/vscode/blob/89659aa4b3fa350e948f3bbc54caf9da8a78be33/build/win32/i18n/messages.tr.isl#L2

File: build/win32/i18n/messages.ru.isl [hunk 2]
```diff
@@ -1,8 +1,9 @@
 [CustomMessages]
-AddContextMenuFiles=�������� �������� "������� � ������� %1" � ����������� ���� ����� ���������� Windows
-AddContextMenuFolders=�������� �������� "������� � ������� %1" � ����������� ���� �������� ����������
-AssociateWithFiles=���������������� %1 � �������� ��������� ��� �������������� ����� ������
-AddToPath=�������� � PATH (�������� ����� ������������)
-RunAfter=��������� %1 ����� ���������
-Other=������:
-SourceFile=�������� ���� %1
\ No newline at end of file
+AddContextMenuFiles=�������� �������� "������� � ������� %1" � ����������� ���� ����� ���������� Windows
```
Comment (jaqra) [inline build/win32/i18n/messages.ru.isl]: Lastly i want to say. It seems your editor changes charsets of files. May this breaks

Comment (jaqra) [reply build/win32/i18n/messages.ru.isl]: Sorry for one by one comment 😕

Comment (hadzhiyski) [reply build/win32/i18n/messages.ru.isl]: I will look into it later today

Comment (hadzhiyski) [reply build/win32/i18n/messages.ru.isl]: I am using VS Code to do the changes, but perhaps the encodings are not ok on my PC

**Review by joaomoreno — CHANGES_REQUESTED** [review]
Yeah, like @jaqra mentions, please fix the encoding issues. :+1: Thanks for the PR!

**Comment by hadzhiyski** [comment]
Translations are added for all supported languages and the encoding issues are fixed. Please review

**Review by joaomoreno — CHANGES_REQUESTED** [review]
@hadzhiyski I still see a lot of changes which don't make sense. Are you maybe converting line endings?

**Comment by hadzhiyski** [comment]
Is that what you mean? I deleted the empty lines on end of files, because that was the original look of the files. I added them now. Perhaps I do not have proper file formatting settings and I am really sorry about this and I will fix it. It is my first PR in vscode repository.

**Comment by joaomoreno** [comment]
Not yet... Notice here, all these lines are marked as changed, yet there is no reason for you to have changed them. There must still be some line-ending/encoding change:

image

File: build/win32/i18n/messages.de.isl [hunk 3]
```diff
@@ -1,8 +1,9 @@
 [CustomMessages]
```
Comment (jaqra) [inline build/win32/i18n/messages.de.isl]: You can try to copy this line's enter char and paste other lines for enter character

Comment (hadzhiyski) [reply build/win32/i18n/messages.de.isl]: I think it's ok now. The huge diff comes from
image

However, <F6> and <C3><B6> is basically the same.
image

Comment (jaqra) [reply build/win32/i18n/messages.de.isl]: May you should set charset as ANSI

Comment (hadzhiyski) [reply build/win32/i18n/messages.de.isl]: i think i managed to fix this

**Comment by joaomoreno** [comment]
Great job, thanks!

**Comment by hadzhiyski** [comment]
Thank you all for the support

**Pull request merged**